content/en/blog/2023/kubecon-na.md

```diff
@@ -0,0 +1,119 @@
+  Gera, Coralogix; Anusha Reddy Narapureddy, Apple<br> Wednesday, November 8 •
+  11:00am - 12:30pm
+- **[Introducing Logs in OpenTelemetry: What It Unlocks and How to Use It](https://sched.co/1R2r4)**<br>
+  by Morgan McLean, Splunk & Dan Jaglowski, observIQ<br> Wednesday, November 8 •
+  2:30pm - 3:05pm
+- **[Jaeger the Future with OpenTelemetry and Metrics](https://sched.co/1R2q1)**<br>
+  by Pavol Loffay, Red Hat<br> Wednesday, November 8 • 3:25pm - 4:00pm
+- **[Tutorial: OpenTelemetry 101: Let's Instrument!](https://sched.co/1R2s9)**<br>
+  by Paige Cruz, Chronosphere<br> Wednesday, November 8 • 4:30pm - 6:00pm
+- **[Remote Control for Observability Using the Open Agent Management Protocol](https://sched.co/1R2sr)**<br>
+  by Jacob Aronoff, Lightstep from ServiceNow & Andy Keller, observIQ<br>
+  Wednesday, November 8 • 5:25pm - 6:00pm
+- **[OpenTelemetry Tracing for Monoliths](https://sched.co/1R2sr)**<br> by
+  Phillip Carter, Honeycomb<br> Thursday, November 9 • 11:00am - 11:35am
+- **[Today, Not Tomorrow: Scalable Strategies for Migrating to OpenTelemetry](https://sched.co/1R2uc)**<br>
+  by Jason Anderson & Kevin Broadbridge, Cruise<br> Thursday, November 9 •
+  2:00pm - 2:35pm
+- **[How and Why You Should Adopt and Expose OSS Interfaces Like OTel & Prometheus](https://sched.co/1R2uc)**<br>
+  by Daniel Hrabovcak & Shishi Chen, Google<br> Thursday, November 9 • 4:55pm -
+  5:30pm
+
+## Co-located Events
+
+[Observability Day][] _fosters collaboration, discussion, and knowledge sharing
+of cloud-native observability projects_. This event will be held on November 6,
+2023 from 9am - 5pm. There will be several sessions on OpenTelemetry as well.
+
+> <i class="far fa-exclamation-triangle"></i> **IMPORTANT access note**: You
+> need an _in-person all-access_ pass for on-site access to **Observability
+> Day**. For details, see [KubeCon registration][]. If you have a virtual
+> ticket, you will be able to follow **Observability Day** through a live
+> stream.
+
+## OpenTelemetry Project Booth
+
+Drop by and say Hi! at the OpenTelemetry project booth in the KubeCon NA Project
+Pavilion. If you’re lucky, you may even pick up some OpenTelemetry swag!
+
+You will find us **TODO**
+
+[Pavilion hours][] are as follows:
+
+- Tuesday, November 7: 10:30am – 8:00pm (includes KubeCrawl + CloudNativeFest
+  from 6:00pm – 8:00pm)
+- Wednesday, November 8: 10:30am – 5:00pm
+- Thursday, November 9: 10:30am – 2:30pm
+
+You can help us improve the project by sharing your thoughts and feedback about
+your OpenTelemetry adoption, implementation, and usage! We also invite you to
+fill out our [community survey][]. We will create action items from your
+comments as appropriate. Check [#otel-user-research][] in CNCF's Slack instance
+for survey results and action item updates to come after KubeCon NA.
+
+Come join us to listen, learn, and get involved in OpenTelemetry.
+
+See you in Chicago!
```

Comment:
Same feeling with exclamation marks.

Reply:
I removed (most) other exclamation marks to keep that one ;)